wp-content/themes/core/routes/unsupported_browser/Unsupported_Browser_Controller.php

```diff
@@ -32,4 +32,14 @@ public function get_legacy_image_url( $filename ): string {
 		return esc_url( trailingslashit( get_template_directory_uri() ) . 'assets/img/theme/legacy-browser/' . $filename );
 	}
 
+	public function get_logo_url(): array {
+		$custom_logo_id = get_theme_mod( 'custom_logo' );
+		$image          = wp_get_attachment_image_src( $custom_logo_id, 'full' );
+		if ( $image ) {
+			return $image;
+		}
+
+		return [];
+	}
+
```

Comment:
We don't use the uploaded logo here b/c it will generally be an SVG and SVGs aren't well supported by unsupported browsers.